app-release-procedures.rst

```diff
@@ -254,8 +210,7 @@ Example 2, db migration:
       * users emails will be exposed
       * authentication policy change - users no longer will be able to log in by `username`
 
-    Roll-out plan
-    -------------
+    ## Roll-out plan
 
     **Deployment risks**:
       * long running DB migration
```

Comment:
We put an app into maintenance mode, users are notified. I don't see this as "risk"